api/conf/schema.json

```diff
@@ -401,6 +398,7 @@
 							"OPTIONS",
 							"PATCH",
 							"POST",
+							"PURGE",
```

Comment:
As far as I can see, this change is not included in the APISIX 2.13 release branch, you need to reproduce it and check it.
https://github.com/apache/apisix/blob/release/2.13/apisix/schema_def.lua#L525-L526